static-assets/components/cstudio-forms/controls/rte.js

```diff
@@ -542,14 +542,16 @@ CStudioAuthoring.Module.requireModule(
             _thisControl.save();
           };
           _thisControl.form.registerBeforeSaveCallback(callback);
+
+          this.renderHelp(config, containerEl);
```

Comment:
help icon/funciontality was not showing on the RTE control

Reply:
The feature was accessible in forms, that's how we found the bug. Where was it not showing? Pls clarify and double check what's affected by this.